cfme/tests/services/test_provision_stack.py

```diff
@@ -1,14 +1,15 @@
 import pytest
 import fauxfactory
 
+from cfme.configure.settings import DefaultViews  # NOQA
 from cfme.services.catalogs.catalog_item import CatalogItem
 from cfme.services.catalogs.catalog import Catalog
 from cfme.services.catalogs.orchestration_template import OrchestrationTemplate
 from cfme.services.catalogs.service_catalogs import ServiceCatalogs
 from cfme.services.catalogs.myservice import MyService
 from cfme.services import requests
-from cfme.web_ui import flash
-from utils import testgen
+from cfme.cloud.stack import Stack
+from utils import testgen, version
 from utils.log import logger
 from utils.wait import wait_for
 
```

Comment:
Comment here refers to the ```METHOD_TORSO``` variable which is hidden

@sshveta Here is my proposal, we talked about there being a conflict in the METHODs and about using the spaces for the "randomness". I'm still not overall happy with this and think it could lead to other issues. However I do come bearing a potential solution.

What if we add a replaceable portion to the ```Description``` parameter of the ```METHOD_TORSO```? Then as we come to use it, we replace it with a random string? This way we are avoiding the fact that we can't use comments and still get to add random data.